platform/migration/platform-migratable-api/src/main/java/org/codice/ddf/migration/MigrationException.java

```diff
@@ -14,37 +14,102 @@
 
 package org.codice.ddf.migration;
 
+import javax.annotation.Nullable;
+
+import org.apache.commons.lang.Validate;
+
 /**
- * Exception that indicates some problem with the configuration migration.
+ * Exception that indicates some problem with the migration operation.
+ * <p>
+ * <i>Note:</i> Detail messages are displayed to the administrator on the console during a
+ * migration operation.
+ * <p>
+ * <b>
+ * This code is experimental. While this class is functional
+ * and tested, it may change or be removed in a future version of the
+ * library.
+ * </b>
+ * </p>
  */
-public class MigrationException extends RuntimeException {
-
-    private static final long serialVersionUID = 1L;
+public class MigrationException extends RuntimeException implements MigrationMessage {
+    private static final long serialVersionUID = -1023839569683504743L;
```

Comment:
Just because, serial version UID should be set to valid values and not changed ever after that. 1L, although a long, is not a valid value for serialization.

Reply:
I agree that as long as the class serialization/deserialization remains compatible, `serialVersionUID` should remain the same. And as long as it's a `long`, it can be anything we want, even `1L`. Which is why I was asking why we were changing it. No big deal though.